### PR TITLE
remove citus from django settings on india

### PR DESCRIPTION
**Pull request by snopoke**
This change should be tested first before rolling out.

##### ENVIRONMENTS AFFECTED
india

### CODE REVIEW

**Comment by rohit-dimagi** [comment]
Maybe we should also remove standby settings from inventory. ?

File: environments/india/postgresql.yml [hunk 1]
```diff
@@ -53,9 +53,10 @@ dbs:
         shards: [820, 1023]
         pgbouncer_host: pgbouncer0
   custom:
-    - django_alias: icds-ucr-citus
-      name: icds-ucr-citus
-      django_migrate: True  # ICDS dashboard models are stored here
+    # remove citus from django
+    # django_alias: icds-ucr-citus
+    # django_migrate: True  # ICDS dashboard models are stored here
```
Comment (dannyroberts) [inline environments/india/postgresql.yml:58]:
```
BadValueError: Property django_migrate is required.
```
looks like it needs to to change this to
```suggestion
    django_migrate: False
```

**Comment by snopoke** [comment]
@rohit-dimagi the inventory should always represent what's actually there. If we are switching the standbys to primaries then we can make that change but in a separate PR

File: environments/india/postgresql.yml [hunk 2]
```diff
@@ -53,9 +53,10 @@ dbs:
         shards: [820, 1023]
         pgbouncer_host: pgbouncer0
   custom:
-    - django_alias: icds-ucr-citus
-      name: icds-ucr-citus
-      django_migrate: True  # ICDS dashboard models are stored here
+    # remove citus from django
+    # django_alias: icds-ucr-citus
+    django_migrate: False
```
Comment (rohit-dimagi) [inline environments/india/postgresql.yml:58]: missing indent.

Comment (snopoke) [reply environments/india/postgresql.yml:58]:
```suggestion
     django_migrate: False
```

**Comment by dannyroberts** [comment]
Came across this just now. @snopoke do you know if the plan's to close this, to merge it (because we just forgot), or to wait until a particular thing that's blocking it?

**Comment by snopoke** [comment]
@dannyroberts it should be safe to roll this once this PR is merged: https://github.com/dimagi/commcare-hq/pull/28532

**Comment by snopoke** [comment]
@dannyroberts actually we want to keep this a bit longer since we may need to use this cluster for testing the citus upgrade.

**Comment by snopoke** [comment]
Rolled out to `india` on 2020/10/20

**Pull request merged**